src/domains/movies.js

```diff
@@ -0,0 +1,92 @@
+const prisma = require("../utils/prisma");
+
+const getAllMoviesDb = async ({ runtimeLt, runtimeGt }) => {
+  runtimeGt = parseInt(runtimeGt);
+  runtimeLt = parseInt(runtimeLt);
+
+  const movieQuery = {
+    include: {
+      screenings: true,
+    },
+  };
+
+  if (runtimeLt) {
+    movieQuery.where = {
+      runtimeMins: {
+        lt: runtimeLt,
+      },
+    };
+  }
+
+  if (runtimeGt) {
+    movieQuery.where = {
+      runtimeMins: {
+        gt: runtimeGt,
+      },
+    };
+  }
```

Comment:
Nice query building. What if you want to have both lt and gt in the same query?

Reply:
It does work as it is I'm pretty sure

Reply:
You were right, I've updated accordingly